gr-uhd/apps/uhd_siggen_gui

```diff
@@ -190,7 +191,8 @@ class uhd_siggen_gui(Qt.QWidget):
         self._freq2_offset_win.setEnabled(self._sg[uhd_siggen.TYPE_KEY] in self._freq2_enable_on)
         self.top_grid_layout.addWidget(self._freq2_offset_win, 4, 3, 1, 2)
         ### Amplitude
-        self._amplitude_range = Range(0, 1, .001, .7, 200)
+        self._amplitude_range = \
+            Range(0, 1, .001, self._sg[uhd_siggen.AMPLITUDE_KEY], 200)
```

Comment:
bottom of AMPL range shoould be not "0" ... but the minimum delta .. so 0.001 ... to avoid the log(0) issue ... yes?

Reply:
See above.